RELEASE.rst

```diff
@@ -0,0 +1,17 @@
+Release Procedure for pyvo
+==========================
+
+These steps are intended to help guide a developer into
+making a new release.  For these instructions, version
+is to be replaced by the version number of the release.
+
+1. Edit setup.cfg and remove .dev from the version number
+2. Edit CHANGES.rst to change unreleased to the date of the release
+Add a new section at the top for the next release number
```

Comment:
nope, do it only after the tag and release is done.

Reply:
okay sounds good I've moved that below into the setup of adding the .dev part back to the version number